MANIFEST.in

```diff
@@ -0,0 +1 @@
+include *.rst
```

Comment:
Do you want all rst-files?

Reply:
It won't add `docs/*` only top level files ending in `.rst`. `sdist` already adds all tests, the README and so on automatically, so I didn't see any issue with adding the `CONTRIBUTORS` and `CONTRIBUTING` files as well.